examples/edgeColoured.g

```diff
@@ -0,0 +1,6 @@
+g1 := NautyEdgeColoredGraph( [[[1,2],[3,10],[11,12],[4,9],[5,6],[7,8]],[[1,12],[2,3],[10,11],[4,5],[6,7],[8,9]],[[3,4],[9,10]]],12);
+g2 := NautyEdgeColoredGraph( [[[3,11],[9,10],[4,7],[1,2],[5,12],[6,8]],[[7,11],[4,10],[3,9],[1,5],[2,8],[6,12]],[[5,9],[10,12]]],12);
+
+IsomorphismGraphs(g1,g2);
+
+Assert( IsomorphismGraphs(g1,g2) = (1,2)(3,5)(4,9,10,12,8)(6,11), "Edge-coloured isomorhism fails");
```

Comment:
To turn this into a test file, basically prefix all input lines with `gap> `. Then either already fill in the expected return values; resp. suppress them using `;;` if they are uninteresting and long. Or let GAP do the work by using `Test("FILENAME.tst", rec(rewriteToFile:=true));`. 

That way I came up with this:
```
gap> g1 := NautyEdgeColoredGraph( [[[1,2],[3,10],[11,12],[4,9],[5,6],[7,8]],[[1,12],[2,3],[10,11],[4,5],[6,7],[8,9]],[[3,4],[9,10]]],12);;
gap> g2 := NautyEdgeColoredGraph( [[[3,11],[9,10],[4,7],[1,2],[5,12],[6,8]],[[7,11],[4,10],[3,9],[1,5],[2,8],[6,12]],[[5,9],[10,12]]],12);;
gap> IsomorphismGraphs(g1,g2);
(1,2)(3,5)(4,9,10,12,8)(6,11)
```